Changes.md

```diff
@@ -1,5 +1,15 @@
 # Medic Mobile Release Notes
 
+## 2.8.6
+
+### TBD
+
+#### Continuous Integration
+
+- Automatically update to master branches of `api` and `sentinel` submodules
+  before running CI in Travis. [#3621](https://github.com/medic/medic-webapp/pull/3621)
+- Fixed kanso packages that inadvertantly cached data. Issue: #3648
+
```

Comment:
Ok well in that case: I don't think CI fixes are relevant in the changelog as the audience doesn't care. It doesn't change the code they get. So I think remove it.